src/pages/Instructor/Instructor.js

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import S from './Styled.Instructor';
```

Comment:
스타일 파일의 리뷰에 따라 해당 import 부분도 변경되어야 합니다.
```suggestion
import * as S from './Styled.Instructor';
```

Reply:
아 저렇게 하면 전체 import가 되는거였군요!! 감사합니다.